### PR TITLE
update docs to use new envs

### PR DESCRIPTION
**Pull request by grantleehoffman**

### CODE REVIEW

**Comment by bkono** [comment]
@grantleehoffman Haven't tested the quickstart myself, but all the renames look consistent and valid to align with the PR that just merged re: prefixes.

File: docs/developers/development-env.md [hunk 1]
```diff
@@ -93,25 +93,25 @@ You will need two windows
...
 
 ### Create Cello Project And Target (One Time Setup)
 
-- In window **#2**, ensure you have the **ARGO_CLOUDOPS_ADMIN_SECRET**
+- In window **#2**, ensure you have the **CELLO_ADMIN_SECRET**
   env set to the same value used above.
 
 - Ensure your credentials are set for the **target account** and create your first
-  project and target. This returns the **ARGO_CLOUDOPS_USER_TOKEN** for the new project.
+  project and target. This returns the **CELLO_USER_TOKEN** for the new project.
```
Comment (Acepie) [inline docs/developers/development-env.md:106]: I don't think user token should be updated to use `CELLO` for the prefix. I think that one is being read by the cli as `ARGO_CLOUDOPS_USER_TOKEN` still

File: scripts/create_project.sh [hunk 2]
```diff
@@ -5,16 +5,16 @@ ACCOUNT_ID=$2
...
 
-if [ -z $ARGO_CLOUDOPS_TARGET_NAME ]; then
-    export ARGO_CLOUDOPS_TARGET_NAME=target1
+if [ -z $CELLO_TARGET_NAME ]; then
+    export CELLO_TARGET_NAME=target1
 fi
 
-if [ -z $ARGO_CLOUDOPS_SERVICE_ADDR ]; then
-    export ARGO_CLOUDOPS_SERVICE_ADDR=https://localhost:8443
+if [ -z $CELLO_SERVICE_ADDR ]; then
```
Comment (Acepie) [inline scripts/create_project.sh:16]: I think this needs to be `ARGO_CLOUDOPS` still? I think this one is only being used by the cli

**Review by Acepie — APPROVED** [review]
LGTM

**Pull request merged**